File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-06/plagiarized/L3/07/Kasus6L3.kt
package IR

import java.util.*

/*
* To change this license header, choose License Headers in Project Properties.
* To change this template file, choose Tools | Templates
* and open the template in the editor.
*/

/**
 *
 * @author B15130F5DDB6B5F1622EF91DAC4C1AAE
 */
object Kasus6L3 {
    @JvmStatic
    fun main(args: Array<String>) {
        //Delarasi
        val arrNum = IntArray(10)
        val inp = Scanner(System.`in`)
        // for untuk 10 kali input
        for (i in 0..9) {
            // Read a number
            print("Read a number: ")
            arrNum[i] = inp.nextInt()
        }
        // Display the array
        for (i in 9 downTo 0) {
            //Print angka dalam array
            println(arrNum[i])
        }
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L3/07/Kasus6L3.py
import sys

# Delarasi
arrNum = [0] * 10
inp = sys.stdin

# for untuk 10 kali input
for i in range(10):
    # Read a number
    print("Read a number: ", end="")
    arrNum[i] = int(inp.readline().strip())

# Display the array
for i in range(9, -1, -1):
    # Print angka dalam array
    print(arrNum[i])

